src/test/java/core/basesyntax/service/impl/DataProcessorImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static core.basesyntax.model.FruitTransaction.Operation.RETURN;
+import static core.basesyntax.model.FruitTransaction.Operation.SUPPLY;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataProcessor;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DataProcessorImplTest {
+    private static final String BANANA_FRUIT = "banana";
+    private OperationStrategy operationStrategy;
+    private DataProcessor dataProcessor;
+    private List<FruitTransaction> fruitTransactions;
+
+    @BeforeEach
+    void setUp() {
+        operationStrategy = new OperationStrategyImpl(Map.of(
+                BALANCE, new BalanceOperationHandler(),
+                RETURN, new ReturnOperationHandler(),
+                PURCHASE, new PurchaseOperationHandler(),
+                SUPPLY, new SupplyOperationHandler()
+        ));
+        dataProcessor = new DataProcessorImpl(operationStrategy);
```

Comment:
```suggestion
        operationStrategy = new OperationStrategyImpl(Map.of(
                BALANCE, new BalanceOperationHandler(),
                RETURN, new ReturnOperationHandler(),
                PURCHASE, new PurchaseOperationHandler(),
                SUPPLY, new SupplyOperationHandler()
        ));
        dataProcessor = new DataProcessorImpl(operationStrategy);
```

Do you actually need to instantiate it before each test, not before all? See the same comments and fix in all such cases.